src/main/java/command/UserInput.java

```diff
@@ -0,0 +1,130 @@
+package command;
+import java.util.Scanner;
+
+import task.Deadline;
+import task.Event;
+import task.Task;
+import task.ToDo;
+
+import java.util.InputMismatchException;
+import java.util.List;
+
+public class UserInput {
+    public static void parseInput(String input, Scanner in, List<Task> taskList) {
+
+        int taskID;
+
+        switch (input.toLowerCase()) {
```

Comment:
unecessary spacing between the cases, and I can see this in other places too. structure code logically and do not add whitespaces where it may not be needed (e.g. right before the closing brace of a function/block, right after an opening brace of a function/block etc.) 

see here: https://nus-cs2113-ay2324s2.github.io/website/se-book-adapted/chapters/codeQuality.html#structure-code-logically